hnn_core/network.py

```diff
@@ -328,6 +328,11 @@ class Spikes(object):
         The inner list contains the type of spike (e.g., evprox1
         or L2_pyramidal) that occured at the corresonding time stamp.
         Each gid corresponds to a type via Network::gid_dict.
+    vsoma : dict
+        Dictionary indexed by gids containing somatic voltages
+    t_vec : list
+        List of time points for samples in continuous data.
+        This includes vsoma.
```

Comment:
any change you make to the object will necessarily break a round trip IO. I know we will fix this in another PR but something to be aware of